chapter01/run_mini_book.md

```diff
@@ -0,0 +1,34 @@
+
+# Running the mini-book code examples
+
+
+Here we suggest the following options to run the code notebooks in this mini-book:
+
+- Running on your local machine (preferred)
+- Running using the launch buttons (for quick exploration)
+
+## Option 1: Running on your local machine (preferred)
+
+You can download each notebook by clicking on the download button at the  top-right corner of the notebook, then run it on your local machine.
+
```

Comment:
Doesn't one need to install CUQIpy to do this? Can you add instructions? Maybe by pointing to the installation instructions

Reply:
Ah I see it is stated below. Maybe move to here, so as to avoid it being missed if you immediately go ahead and download the notebook.

Reply:
good point, I updated the instructions